tasks/transform_conformance/oxc.snap.md

```diff
@@ -1,13 +1,19 @@
 commit: 12619ffe
 
-Passed: 10/36
+Passed: 10/37
 
 # All Passed:
 * babel-plugin-transform-optional-catch-binding
 * babel-plugin-transform-arrow-functions
 
 
-# babel-plugin-transform-typescript (2/7)
+# babel-plugin-transform-typescript (2/8)
+* class-property-definition/input.ts
+  x Unresolved references mismatch:
+  | after transform: ["const"]
+  | rebuilt        : []
+
+
```

Comment:
There seems to be an incorrect check here. The `const` is an unresolved reference, after transformation, the `const` will be removed, Do we need to remove it from `unresolved_references`? @overlookmotel cc

Reply:
I don't know :-)

Reply:
```ts
class Foo {
  #handlers = [] as const;
}
```

`const` in the original TS AST is an `IdentifierReference` which has no binding, so it's recorded in `unresolved_references`. When we remove the `IdentifierReference` in transform, we should remove it from `unresolved_references` too.